provision/setup-montagu

```diff
@@ -56,8 +53,6 @@ else
     pip3 install --user -r ${MONTAGU_PATH}/src/requirements.txt
     cp ${MONTAGU_PATH}/settings/${CONFIG_NAME}.json \
        ${MONTAGU_PATH}/src/montagu-deploy.json
-    # NOTE: if we ever run deploy here we need to be careful about
-    # sudo, groups and docker membership.
 fi
 
```

Comment:
We can also add command to create bb8 symlink here

Reply:
ok, I can't test this until https://github.com/vimc/montagu-bb8/pull/7 is resolved, but I've made the code change

Reply:
"Works On My Machine"